### PR TITLE
Fixed install command

### PR DESCRIPTION
**Pull request by G3bE**
# Pull Request Template

Please try and complete the below as best as possible. Some of the fields may
not be necessary so feel free to add or edit as you see fit.

## Checklist

_Confirm you have completed the following actions prior to submitting this PR._

- [ ] There is an existing issue report for this PR.
- [x] I have forked this project.
- [x] I have created a feature branch.
- [x] My changes have been committed.
- [ ] My changes were linted using pylava.
- [x] I have pushed my changes to the branch.

## Title

Fixed install command

## Description

The install command for this branch was still the one as in `dev`

## Issue Resolution

This Pull Request Fixes the install command

### CODE REVIEW

**Comment by itsVale** [comment]
Please also create a proper example on how to use the dev-gateway branch. Then I will merge it.

File: README.md [hunk 1]
```diff
@@ -30,14 +30,23 @@ and the Gateway, then there will be docs added.
...
+def on_message(event):
+    # Event will become an object soon. For now, it's just the raw response dict from the Discord API
+    channel = event['channel']
+
+    if event['content'].startswith('!ping'):
+        client.api.create_message(channel, 'Pong')
+
+client.start('Token')  # get your token from Discords application page: https://discordapp.com/developers/applications/
 ```
 There is no way to receive events.
```
Comment (romangraef) [inline README.md]: Well, two lines above you state how to use events.

**Comment by G3bE** [comment]
Hope I fixed everything

File: README.md (hunk 1 above)
Comment (G3bE) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -18,26 +18,34 @@
 _Though this library is incomplete yet, contributions are very appreciated!_
 
 # Installation
-For now, you can only install the dev branch which is constantly updated.
 ```
-pip install -U https://github.com/itsVale/Shitcord/archive/dev.zip
+pip install -U https://github.com/itsVale/Shitcord/archive/dev-gateway.zip
 ```
```
Comment (romangraef) [inline README.md]: We wont leave this on the gateway branch forever. For now this is fine tho.

Comment (G3bE) [reply README.md]: Hm. Wouldn't it be better if someone who wants to use this branch could install it right away without changing the command by himself?

Comment (romangraef) [reply README.md]: We ofc could publish this package on pypi. But we aren't in a production ready state yet.

**Comment by itsVale** [comment]
I think this isn't necessary anymore. Just created a PR that merges the dev-gateway branch into dev so there's no need for the changed installation link and also I think it would be better in general to leave logging away in that simple example.

**Pull request closed without merging**